ek/README.md

```diff
@@ -0,0 +1,28 @@
+# The EFK stack
```

Comment:
Why the folder is `ek` and everywhere EFK ?
Do you think we can name the folder `monitoring` ?

Reply:
ek was because fluentd is not created in this directory but in workers. 
I agree that we can call it `monitoring`